_i18n/pl/license.md

```diff
@@ -0,0 +1,1188 @@
+---
+title: "Licencja"
+lang: "pl"
+layout: qmk
+permalink: /pl/license/
+---
+
+# Licencja
+
+Większość samego QMK jest objęta licencją GPL v2, ale zawiera oprogramowanie objęte różnymi innymi licencjami, z których wszystkie są krótko opisane we wnętrzu tego pakietu. Pełny tekst każdej licencji jest dostępny w repozytorium kodu i poniżej.
+
+The MIT License (MIT)
```

Comment:
Would you mind creating this file again? GitHub seems to still be struggling with this file's encoding and/or contents (see the "quotes" on Lines 17, 27, etc.):

![image](https://user-images.githubusercontent.com/18669334/95720601-50dec780-0c26-11eb-8afc-2c90adedd7ba.png)